georaster-layer-for-leaflet.js

```diff
@@ -442,6 +446,56 @@ const GeoRasterLayer = L.GridLayer.extend({
     }
   },
 
+  /**
+   * The callback used to determine the colour based on the values of each pixel
+   *
+   * @callback pixelValuesToColor
```

Comment:
thank you for adding function documentation!  I think it'll help developers a lot with VSCode and other IDE's intellisense and type ahead.

Reply:
No worries.